src/vivarium_public_health/risks/implementations/low_birth_weight_and_short_gestation.py

```diff
@@ -152,17 +152,26 @@ def single_axis_ppf(
     def _parse_description(axis: str, description: str) -> pd.Interval:
         """
         Parses a string corresponding to a low birth weight and short gestation
-        category to an Interval
+        category to an Interval.
+        An example of a standard description:
+        'Neonatal preterm and LBWSG (estimation years) - [0, 24) wks, [0, 500) g'
+        An example of an edge case for gestational age:
+        'Neonatal preterm and LBWSG (estimation years) - [40, 42+] wks, [2000, 2500) g'
+        An example of an edge case of birth weight:
+        'Neonatal preterm and LBWSG (estimation years) - [36, 37) wks, [4000, 9999] g'
+
         :param axis:
         :param description:
         :return:
         """
         endpoints = {
             BIRTH_WEIGHT: [
-                float(val) for val in description.split(", [")[1].split(")")[0].split(", ")
+                float(val)
```

Comment:
Not sure how you ended up w/ this from Hussain's PR, but if this string parsing had a bug then that implies that the `_parse_description` unit tests were lacking